src/Composition/massPropertiesComputation.jl

```diff
@@ -74,13 +74,14 @@ end
 
 # function computeInertiaTensorForTwoBodies!(massPropParent, massPropObj, obj; add=true)
 function addOrSubtractMassPropertiesOfChildToRoot!(obj_root::Object3D{F}, obj_child::Object3D{F}; add=true)::Nothing where F <: Modia3D.VarFloatType
-    massProperties_child = obj_child.feature.massProperties
+    solid::Shapes.Solid{F} = obj_child.feature
+    massProperties_child::Shapes.MassProperties{F} = solid.massProperties
 
     R_child   = obj_child.R_rel
     r_child   = obj_child.r_rel
-    m_child   = massProperties_child.m
-    rCM_child = massProperties_child.rCM
-    I_child   = massProperties_child.I
+    m_child::F                = massProperties_child.m
+    rCM_child::SVector{3,F}   = massProperties_child.rCM
+    I_child::SMatrix{3,3,F,9} = massProperties_child.I
```

Comment:
Type definition is not necessary.

Reply:
I'm not definitely sure, if line 78 is enough. Therefore I'd need to re-run it.

Reply:
Type definitions aren't needed, `massProperties_child::Shapes.MassProperties{F} = solid.massProperties` is sufficient